bpdm-cleaning-service-dummy/src/main/resources/application.yml

```diff
@@ -29,27 +29,34 @@ bpdm:
     # Connection to Orchestrator
     client:
         orchestrator:
-            base-url: http://localhost:8085/
+          base-url: http://localhost:8085/
 
-    # Logging Configuration
+  # Logging Configuration
     logging:
-        unknown-user: Anonymous
+      unknown-user: Anonymous
 
-    # Cleaning Service Configurations
+  # Cleaning Service Configurations
     cleaningService:
-        pollingCron: '-'
+      pollingCron: '-'
+
+# Logging Configuration
+logging:
+  unknown-user: Anonymous
+  pattern: "%clr(%d{yyyy-MM-dd HH:mm:ss.SSS}){faint} %clr(%5p) %clr([%-20.20X{user:-System}]){blue} %clr([%-36.36X{request:-No Request}]){magenta} %clr([%-15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %m%n%wEx"
+  level:
+    root: DEBUG
```

Comment:
But DEBUG on root level would mean all messages are DEBUG on default, right? How about making root INFO and make the BPDM package name DEBUG?

Reply:
Did the change. Would the logging level's be something like that?